### DISCUSSION TITLE
Issue Connecting to SharePoint site using Connect-PnPOnline -Interactive

### DISCUSSION THREAD

**Discussion by MatthewLindseyFerra** [pnp/powershell#959] [Q&A]
Hello, I am trying to connect to a SharePoint site in my GCCHigh tenant using the -Interactive switch.  It presents me with a login window where I enter my credentials but then that login window switches to an error saying "Can't reach this page" and localhost:XXXXX can't be reached.  I also get an error in the Powershell console that it is not a GET query (see attached screenshot).  

Please note, when I use the -UseWebLogin switch that works and allows me to connect however I don't have as many options and get 403 forbidden errors when trying to download JSON data from a list.  Am I doing something wrong or is this yet another limitation in GCCHigh?  Appreciate any help the community can provide.  

![PnP PowerShell Connect Issue 7-30-21](https://user-images.githubusercontent.com/88197068/127661233-ead11327-1558-47bf-9ff5-8f8a34b8a362.png)

**Comment by LeonArmston** [comment]
Hi Matthew

Here is the guidance re connecting to GCC & some further information:
https://pnp.github.io/powershell/articles/authentication.html#authentication-to-gcc-or-national-cloud-environments

Thanks

Leon

**Comment by heinrich-ulbricht** [comment]
@MatthewLindseyFerra Hi, it's been a while but I face the same error situation as you. Could you resolve your issue? How?

**Comment by heinrich-ulbricht** [comment]
Lol, Google again surfaced this discussion and I see I already commented. Again @MatthewLindseyFerra if you read this, any information would be appreciated.

**Comment by AlexFilipovici** [comment]
Any update on this?
